app/assets/stylesheets/theme/custom.css.scss

```diff
@@ -27,4 +25,20 @@ select {
     float: left;
     width: 50%;
   }
+}
+
+.search-result {
+
+  #search_query {
```

Comment:
Avoid using id selectors<br>Selector `search_query` should be written in lowercase with hyphens